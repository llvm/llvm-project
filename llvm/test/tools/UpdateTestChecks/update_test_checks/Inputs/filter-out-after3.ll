; RUN: opt -S -mtriple aarch64-linux-gnu -passes=instnamer < %s | FileCheck %s

define i32 @f_0(i32) {
  %2 = add i32 %0, 2
  br label %3

; <label>:3:
  ret i32 %2
}
