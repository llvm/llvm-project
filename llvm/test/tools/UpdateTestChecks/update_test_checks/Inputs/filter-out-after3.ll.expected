; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --filter-out-after "br label %3" --version 5
; RUN: opt -S -mtriple aarch64-linux-gnu -passes=instnamer < %s | FileCheck %s

define i32 @f_0(i32) {
; CHECK-LABEL: define i32 @f_0(
; CHECK-SAME: i32 [[ARG:%.*]]) {
; CHECK-NEXT:  [[BB:.*:]]
; CHECK-NEXT:    [[I:%.*]] = add i32 [[ARG]], 2
; CHECK-NEXT:    br label %[[BB1:.*]]
; CHECK:       [[BB1]]:
; CHECK-NEXT:    ret i32 [[I]]
;
  %2 = add i32 %0, 2
  br label %3

; <label>:3:
  ret i32 %2
}
